cloud/spawn.go

```diff
@@ -38,6 +38,8 @@ type SpawnOptions struct {
 	InstanceTags     []host.Tag
 	InstanceType     string
 	NoExpiration     bool
+	AttachVolume     bool
+	HomeVolumeSize   int
```

Comment:
what units is this?

Reply:
GB.
I changed the variable name and added a comment to make it clearer.